### ISSUE TITLE
NTR: CT or MRI: pancreatic mass (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8904]
New term request CT or MRI: pancreatic mass (elthat_100319044519):   UMLS: 0877425

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000409
- label: Pancreatic mass
- parents: [Abnormal pancreas morphology (HP:0012090)](https://hpo.jax.org/app/browse/term/HP:0012090)
- pmid: 
- def: A swelling or enlargment localized to the mass. The word mass is usually used at an early stage of the diagnostic workup before the precise nature of the swelling has been determined.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8904
- synonyms:
